### ISSUE TITLE
Anyway to display wifi strength and fan speed?

### ISSUE THREAD

**Issue by Ellevix** [TheLastProject/lovelace-valetudo-map-card#120]
Hello,
I was wondering if there is anyway to display the wifi strength and fan speed, and associated icons, in addition to the status and battery level?
Thanks, Cam

**Comment by TheLastProject** [comment]
Should definitely be possible as that info seems to be exposed to Home Assistent by Valetudo.

**Comment by Ellevix** [comment]
Great. Oh and also, is there a way I can have your card display sensor information, eg. the main filter time, brush time, etc.?

**Comment by TheLastProject** [comment]
As those are also exposed to Home Assistant adding those to the card's code should also be relatively easy.

**Comment by Ellevix** [comment]
Thank you :)  Would you have any example code to add the sensor entities to the card that I can try?

**Comment by TheLastProject** [comment]
Not really, but it should mostly just going to be a copy-paste from other stuff. I have too many projects already and I want more contributors/maintainers for this so I'm purposefully avoiding doing everything myself :)

AFAICT quickly without spending too much time on this it'd just be a few new `get*Entity` functions and some new ifs similar to https://github.com/TheLastProject/lovelace-valetudo-map-card/blob/f9134ee2f6e1f346ba9e048c0212ac8bfad809e6/valetudo-map-card.js#L570-L581
